### ISSUE TITLE
Grapher issue: Error: <gvedit>: syntax error in line 3 near ','

### ISSUE THREAD

**Issue by chansey97** [google/guice#1139]
I used Guice's grapher to print .dot file (which dump dependency graph from xtext), and attempt to use graphviz to open it.
But the error occured:
Error: <gvedit>: syntax error in line 3 near ','

I installed graphviz-2.38.0, use guice com.google.inject_3.0.0.v201605172100.jar and guice-grapher-3.0.
The .dot file looks good, is it possible because I am using the wrong GVEDIT version?
Does anyone have the same problem as me? Or anyone can open this attachment file
Thanks.
[MyDslGraph.zip](https://github.com/google/guice/files/1504039/MyDslGraph.zip)

**Comment by codesim** [comment]
Yes, this problem also occurs to me. For me, the problem was in the margin parameters where commas were set incorrectly. At least in small files that could be fixed by hand.
